src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1086,7 +1112,6 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 				block.particleCondition = new Procedure("condition4");
 			}
 			block.itemTexture = emptyLists ? "" : "itest";
-			block.particleTexture = emptyLists ? "" : "test7";

```

Comment:
Probably erroring due to the removal of this line. Is there a specific reason for this one?

Reply:
Yes, when I merged the master branch in this branch.

Reply:
Seems you don't have the same master as upstream master, this line should be there